src/layout/BaseLayout.js

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect, useState } from "react";
+import { Outlet } from "react-router-dom";
+import styles from "../styles/BaseLayout.module.css";
+import { getRequestApi } from "../utils/requestApi";
+import { useRequest } from "../hooks/useRequest";
+
+const Header = () => {
+  const [isSignin, setIsSignin] = useState({});
```

Comment:
r: `is~`는 참/거짓을 묻는 질문이라 보통 `boolean` 값을 저장하는 변수명으로 많이 쓰여요. 로그인 여부를 나타내신 거라면 `boolean` 값을 넣는 상태가 적절해보입니다!
로그인한 유저 정보는 별도의 상태에 저장하면 역할이 명확해 쉽게 구분하여 사용할 수 있을 것 같아요.

```suggestion
  /**
   * 로그인 여부
   */
  const [isSignin, setIsSignin] = useState(false);
  /**
   * 유저 정보
   */
  const [user, setUser] = useState();
```